lgc/builder/BuilderImpl.h

```diff
@@ -286,7 +286,7 @@ class DescBuilder : virtual public BuilderImplBase {
   llvm::Value *CreateLoadPushConstantsPtr(llvm::Type *pushConstantsTy, const llvm::Twine &instName) override final;
 
   // Create a buffer length query based on the specified descriptor.
-  llvm::Value *CreateGetBufferDescLength(llvm::Value *const bufferDesc,
+  llvm::Value *CreateGetBufferDescLength(llvm::Value *const bufferDesc, unsigned offset,
```

Comment:
Can we pass a Value instead of an unsigned here please? It is then more consistent with most other Builder APIs.

Reply:
Fixed.